README.md

```diff
@@ -84,7 +96,10 @@ Define a global variable instructor and set it equal to your Squad Instructor's
 
 Your Answer:
 ```js
-// write code here
+var instructor = "becky";
```

Comment:
Yes, line 99 is a good example of a local variable. For this question, I was just getting at showing examples between global and local variables. For example: 
```
// global variable
instructor = "Becky";
// local variable
var instructor = "Becky";
```